gradle.properties

```diff
@@ -1,8 +1,7 @@
 org.gradle.caching=true
 puppycrawlCheckstyleVersion=8.18
 group=org.ballerinalang
-version=0.6.0-alpha6-SNAPSHOT
```

Comment:
do we need to remove this?

Reply:
added back. I saw `SNAPSHOT` and got confused.